cardano-cli/src/Cardano/CLI/Compatible/Transaction/Option.hs

```diff
@@ -62,7 +62,7 @@ pTxInOnly =
   Opt.option
     (readerFromParsecParser parseTxIn)
     ( Opt.long "tx-in"
-        <> Opt.metavar "TX-IN"
+        <> Opt.metavar "TX_IN"
```

Comment:
Can you create an ADR surrounding this change? I think this is the 3rd time we're changing this.

Reply:
Created issue here: https://github.com/IntersectMBO/cardano-cli/issues/1148

Reply:
Removed it from this commit: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/513622cc826f0f7cce3e90ec8a57af83f797368e